gulpfile.js

```diff
@@ -13,20 +14,23 @@ var yargs = require('yargs');
 var path = require('path');
 var htmllint = require('gulp-htmllint');
 var typescript = require('gulp-typescript');
-var typedoc = require("gulp-typedoc");
+var typedoc = require('gulp-typedoc');
 
 var pkg = require('./package.json');
 var tsProject = typescript.createProject('./tsconfig.json');
+var tsBuild = typescript.createProject('./tsconfig.build.json');
```

Comment:
Do we need a new one? Or will it work to just put the new options in the existing `tsconfig.json`?

Reply:
I think it would work. 
This way we have linting and building separated, but not sure how useful it is.